src/views/bottle_preferences.py

```diff
@@ -55,6 +56,7 @@ class PreferencesView(Adw.PreferencesPage):
     btn_manage_gamescope = Gtk.Template.Child()
     btn_manage_sandbox = Gtk.Template.Child()
     btn_manage_versioning_patterns = Gtk.Template.Child()
+    btn_manage_vmtouch = Gtk.Template.Child()
```

Comment:
You may want to make the switcher non sensitive if vmtouch is not present, just by using the check mentioned in the previous comment:

https://github.com/bottlesdevs/Bottles/blob/3ef4024fa2213abba405485aa5ac07fcff4c63b1/src/views/bottle_preferences.py#L213

Reply:
But vmtouch would be installed in the flatpak since I added it to the manifest

Reply:
i just remembered that bottles also has packages other than the flatpak, I'll change the code according to your suggestion